superset-frontend/src/dashboard/stylesheets/dnd.less

```diff
@@ -111,6 +111,7 @@
   & > .empty-droptarget:last-child {
     height: 12px;
     bottom: 0px;
+    visibility: hidden;
```

Comment:
Hard to tell from the screenshots what's being hidden here. But if it's _always_ hidden (which it looks like may be the case) then should it just be `display: none` and remove the `height`/`bottom` attributes if they're not necessary anymore?

Reply:
It's good that you noticed that. With z-index it's not needed anymore and I thought I removed this commit. Now it's removed and I tested it - it works. Sorry for that!